scripts/tools/zap/tests/outputs/all-clusters-app/app-templates/CodedrivenCallback.h

```diff
@@ -0,0 +1,97 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+// THIS FILE IS GENERATED BY ZAP
+// This file is generated from codedriven-callback.h.zapt
+
+// Prevent multiple inclusion
+#pragma once
+
+#include <app/util/basic-types.h>
+
+/**
+ * @param endpoint    Endpoint that is being initialized
```

Comment:
I would consider removing these comments: they make the files larger and it seems obvious that a call of `FooInit(endpointId)` initializes foo on the given endpoint id.